migrations/migration_021.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO users
```

Comment:
Doesn't need to be in test code either, I fixed it the right way awhile ago but forgot to delete the temporary workaround.